src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsPreview.java

```diff
@@ -0,0 +1,139 @@
+  @Test(groups = {"ChatPreview_001", "ChatPreview"})
+  public void ChatPreview_001_twoUserEnterChat() {
+    switchToWindow(driverOne);
+    ChatPageObject chatUserOne = openChatForUser(
+        driverOne, userStaff, userStaffPassword
+    );
+    chatUserOne.verifyChatPage();
+
+    switchToWindow(driverTwo);
+    ChatPageObject chatUserTwo = openChatForUser(
+        driverTwo, userStaff2, userStaffPassword2
+    );
+    chatUserTwo.verifyChatPage();
+    switchToWindow(driverOne);
+
+    chatUserOne.verifyUserJoinToChatMessage(userStaff2);
+  }
+
+
+  @Test(groups = {"ChatPreview_002", "ChatPreview"})
+  public void Chat_002_verifySwitchingBetweenMainAndPrivateSections() {
+    switchToWindow(driverOne);
+    ChatPageObject chatUserOne = openChatForUser(
+        driverOne, userStaff, userStaffPassword
+    );
+
+    switchToWindow(driverTwo);
+    ChatPageObject chatUserTwo = openChatForUser(
+        driverTwo, userStaff2, userStaffPassword2
+    );
+
+    String userTwoMessage = chatUserTwo.generateMessageFromUser(userStaff2);
+    chatUserTwo.writeOnChat(userTwoMessage);
+
+    switchToWindow(driverOne);
+    chatUserOne.verifyMessageOnChat(userTwoMessage);
+
+    chatUserOne.selectPrivateMessageToUser(userStaff2);
+    chatUserOne.verifyPrivateMessageHeader();
+    chatUserOne.verifyPrivateMessageIsHighlighted(userStaff2);
+    chatUserOne.verifyPrivateChatTitle();
+
+    chatUserOne.clickOnMainChat();
+    chatUserOne.verifyMainChatIsHighlighted();
+    chatUserOne.verifyMessageOnChat(userTwoMessage);
+  }
+
+  @Test(groups = {"ChatPreview_003", "ChatPreview"})
+  public void Chat_003_sendPrivateMessage() {
+    switchToWindow(driverOne);
+    ChatPageObject chatUserOne = openChatForUser(
+        driverOne, userStaff, userStaffPassword
+    );
+
+    switchToWindow(driverTwo);
+    ChatPageObject chatUserTwo = openChatForUser(
+        driverTwo, userStaff2, userStaffPassword2
+    );
+
+    String userTwoPublicMessage = chatUserTwo.generateMessageFromUser(userStaff2);
+    chatUserTwo.writeOnChat(userTwoPublicMessage);
+
+    switchToWindow(driverOne);
+    chatUserOne.verifyMessageOnChat(userTwoPublicMessage);
+
+    switchToWindow(driverTwo);
+    String userTwoPrivateMessage = chatUserTwo.generateMessageFromUser(userStaff2);
+    chatUserTwo.selectPrivateMessageToUser(userStaff);
+    chatUserTwo.writeOnChat(userTwoPrivateMessage);
+
+    switchToWindow(driverOne);
+    chatUserOne.verifyPrivateMessageHeader();
+    chatUserOne.verifyPrivateMessageNotification();
+    chatUserOne.clickOnUserInPrivateMessageSection(userStaff2);
+    chatUserOne.verifyMessageOnChat(userTwoPrivateMessage);
+  }
+
+  @Test(groups = {"ChatPreview_004", "ChatPreview"})
+  public void ChatPreview_004_basicUserChatFails() {
+
+    switchToWindow(driverOne);
+    ChatPageObject chatUserOne = openChatForUser(
+            driverOne, userOne, userOnePassword
+    );
+    WebElement permissionError = driverOne.findElement(By.cssSelector(".WikiaPageHeader"));
```

Comment:
Where is the permission error displayed at? Is there a page object that this .WikiaPageHeader belongs to?